tests/test_sdr.py

```diff
@@ -147,8 +147,10 @@ def test_sdr_validation_watershed_missing_ws_id(self):
         self.assertTrue(
             validate_result,
             'expected a validation error but didn\'t get one')
-        self.assertTrue(
-            'Fields are missing from the first layer' in validate_result[0][1])
+        expected = [(
+            ['watersheds_path'],
+            'ws_id matched 0 headers, expected at least one')]

```

Comment:
Same here about the message phrasing.

Reply:
see https://github.com/natcap/invest/pull/526#discussion_r652766326